packages/trie/test/proof/range.spec.ts

```diff
@@ -141,8 +141,8 @@ describe('simple merkle range proofs generation and verification', () => {
     }
 
     // Special case, two edge proofs for two edge key.
-    const startKey = hexStringToBytes('00'.repeat(32))
-    const endKey = hexStringToBytes('ff'.repeat(32))
+    const startKey = prefixedHexStringToBytes('0x' + '00'.repeat(32))
```

Comment:
zeros util

Reply:
I think in this case it is cleaner to keep it like this, since `endKey` uses `prefixedHexStringToBytes` and `startKey` would now use `zeros` - makes it slightly harder to read (I do agree with block tests though these should+will be changed in 25f29dc9fbbbdf3d75b080bb9803e8a6eb97f7ad)